.github/workflows/docs.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         run: cargo build --release
       - run: cargo install rustdoc-stripper
       - run: python3 ./generator.py --embed-docs --yes ./
+      - run: git clone https://gitlab.gnome.org/World/Rust/gir-rustdoc/ # checkout action doesn't support random urls
```

Comment:
Would have make maybe more sense to put it in the release script? Well, not a blocker in any case.

Reply:
No, because it's used by gstreamer and in the future all the crates in World/Rust

Reply:
Oh I see. Well then all good. :)